### PR TITLE
Add banner for KubeCon China 2024

### LINKED ISSUE #46039: Add banner for KubeCon 2024 China
**This is a Feature Request**

**What would you like to be added**

Set a banner to announce KubeCon 2024 China https://events.linuxfoundation.org/kubecon-cloudnativecon-open-source-summit-ai-dev-china/

**Why is this needed**

We usually turn on a banner for the upcoming KubeCon about 4 weeks ahead of the event.
 - for EU and NA, we did several times for 4 or 5 weeks.
 - the first time for KubeCon China is the 2021 virtual KubeCon China, we have the banner for 2 weeks. 2-4 weeks are OK for me. 

Logos are not available yet: https://github.com/cncf/artwork/tree/main/other/kubecon-cloudnativecon/.

https://events.linuxfoundation.org/kubecon-cloudnativecon-open-source-summit-ai-dev-china/
 Also see the [banner](https://github.com/kubernetes/website/blob/main/data/announcements/scheduled.yaml#L96-L105) config from a previous year's KubeCon(KubeCon China 2021 for 2 weeks).

**Comments**
This change will need @kubernetes/steering-committee approval.

I would like to set the banner to start on the July 26th, 2024 and to end on the August 23, 2024 at 1800 CST (UTC+8).

### PR DESCRIPTION
**Pull request by steve-hardman**
I have added banner for KubeCon China 2024 as requested in #46039

Screenshot of the banner 
image

This will resolve #46039

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle Add banner for KubeCon China 2024

File: data/announcements/scheduled.yaml
Comment (dipesh-rawat) [inline data/announcements/scheduled.yaml:237]: The PR number in the comment will need to be updated to reflect the correct number for this pull request.

```suggestion
  startTime: 2024-07-26T00:00:00  #Added in https://github.com/kubernetes/website/pull/46431
```

File: data/announcements/scheduled.yaml
Comment (dipesh-rawat) [inline data/announcements/scheduled.yaml:247]: It looks like the hyperlink params might need to be updated to align with the current China event, as there are references to old KubeCon EU event. I'm unsure of the correct values to use here.

**Review by dipesh-rawat — COMMENTED** [review]
Could you kindly include a screenshot of the banner in the pull request [description](https://github.com/kubernetes/website/pull/46431#issue-2303695773)? This would significantly improve clarity and offer reviewers a helpful visual reference.

**Comment by dipesh-rawat** [comment]
/committee steering

This change will need steering committee approval.

File: data/announcements/scheduled.yaml
Comment (cjyabraham) [reply data/announcements/scheduled.yaml:247]: I think for now just change `kubeconeu24` to `kubeconcn24`. The rest should be correct.

Comment (pacoxu) [reply data/announcements/scheduled.yaml:247]: https://events.linuxfoundation.org/kubecon-cloudnativecon-open-source-summit-ai-dev-china/register/?utm_source=kubernetes&utm_medium=all&utm_campaign=kubeconcn24&utm_content=slim-banner

The correct link IIUC.

**Review by soltysh — APPROVED** [review]
/lgtm 
steering

**Comment by mrbobbytables** [comment]
/lgtm
with steering hat on :+1:

**Comment by pohly** [comment]
/lgtm

Steering.

**Comment by justaugustus** [comment]
+1 from Steering, but:
/hold to fix the UTM parameters

**Review by pacoxu — APPROVED** [review]
/lgtm
Steering.

Thanks, everyone.

**Pull request merged**
